### ISSUE TITLE
`-v` flag should include logs about incoming/outgoing chatwoot messages

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2990]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "minor"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.54.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.53.0...4.54.0) (2022-12-24)

- ⬆️ upgrade pptr deps #2988 [`a236682`](https://github.com/open-wa/wa-automate-nodejs/commit/a236682fc0b26c01e96fe912d7a5ec457f7882b9)
- ⬆️ upgrade socketio deps #2988 [`3e40665`](https://github.com/open-wa/wa-automate-nodejs/commit/3e4066576a9b0f28c1f903a6436cb24d9bac7c3e)
- 📝 docs update [`c47da60`](https://github.com/open-wa/wa-automate-nodejs/commit/c47da6022235bc2b981ff65a9e647a240e899745)
- Release 4.54.0 [`59c9343`](https://github.com/open-wa/wa-automate-nodejs/commit/59c9343ad7b4264b90a71fda4cdd4ac25d15663d)
- 🔨 CLI: `--tunnel` improvements. #2989 [`40aa9e3`](https://github.com/open-wa/wa-automate-nodejs/commit/40aa9e3a06727821541a4753e6e508f13dacc29f)
- 🔊 log kill reason [`d7ad94b`](https://github.com/open-wa/wa-automate-nodejs/commit/d7ad94b5af3a809965a495060d0e0bdbd973f9fe)
- 🔊: added more chatwoot verbose logs #2990 [`3ababdc`](https://github.com/open-wa/wa-automate-nodejs/commit/3ababdc5f9015f5c79a372333fa5d7962bde810e)
- 🔇 remove `EXPERIMENTAL` md logs #2986 [`a1daed9`](https://github.com/open-wa/wa-automate-nodejs/commit/a1daed970b93ea1a7699030302c11080a112aee1)
- updated types-only package [`de76a92`](https://github.com/open-wa/wa-automate-nodejs/commit/de76a923202a6447483ab9bd171d9c5a3d63c1f7)
- 💚 fix ts error during pptr upgrade #2988 [`6ab90a0`](https://github.com/open-wa/wa-automate-nodejs/commit/6ab90a049b79f8684deea6aef22cf3e64f6dfaad)
